release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.IO.Pipes.AccessControl.md

```diff
@@ -0,0 +1,15 @@
+# System.IO.Pipes.AccessControl
+
+``` diff
+ {
+     namespace System.IO.Pipes {
++        public static class AnonymousPipeServerStreamAcl {
++            public static AnonymousPipeServerStream Create(PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity);
++        }
++        public static class NamedPipeServerStreamAcl {
++            public static NamedPipeServerStream Create(string pipeName, PipeDirection direction, int maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, int inBufferSize, int outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability = 0, PipeAccessRights additionalAccessRights = );
```

Comment:
Do we need to indicate nullability in these diffs? Because `PipeSecurity pipeSecurity` should be `PipeSecurity? pipeSecurity`.

Also, the last parameter, `PipeAccessRights additionalAccessRights =` should've had `default` as the _default_ value.
[runtime/src/libraries/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.cs#L76](https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.cs#L76)

Reply:
default seems like a bug, we have not been adding nullability changes in the diff, that would add a lot of noise